File: crates/ctxctl/tests/fixtures/sample.cs
using System;
using System.Collections.Generic;
using static System.Math;

namespace Demo.App {
    /// A 2D point.
    public class Point {
        private double x;
        private double y;

        /// Distance from the origin.
        public double Norm() {
            return Sqrt(x * x + y * y);
        }
    }

    public interface IRepo {
        List<string> All();
    }

    public record Pair(int A, int B);

    public enum Status { On, Off }

    public struct Vector2 {
        public double X;
    }
}

File: crates/ctxctl/tests/fixtures/sample.java
package com.example.app;

import java.util.List;
import static java.lang.Math.PI;
import com.example.util.Helper;

/** A 2D point. */
public class Point {
    private double x;
    private double y;

    /** Distance from the origin. */
    public double norm() {
        return Math.sqrt(x * x + y * y);
    }
}

/** A repository interface. */
public interface Repo {
    List<String> all();
}

public class App {
    public static void main(String[] args) {}
}
